rts/lra/run.sh

```diff
@@ -1,4 +1,16 @@
 #!/usr/bin/env bash
+
+
+if [[ ! -v WORKSPACE ]]; then
+    echo "Please set the WORKSPACE variable to the root folder of the quickstart"
+    exit -1
+fi
+
+if [ ! -f $WORKSPACE/rts/lra-examples/coordinator-quarkus/target/lra-coordinator-quarkus-runner.jar ]; then
+    echo "Please build first the coordinator-quarkus module which is needed for this demo"
```

Comment:
I think it is the `lra-coordinator-quarkus` module? It might be worth mention that this is in $WORKSPACE/rts/lra-examples